staging/src/k8s.io/apiserver/pkg/storage/etcd3/lease_manager.go

```diff
@@ -79,6 +109,9 @@ func (l *leaseManager) GetLease(ctx context.Context, ttl int64) (clientv3.LeaseI
 	// cache the new lease id
 	l.prevLeaseID = lcr.ID
 	l.prevLeaseExpirationTime = now.Add(time.Duration(ttl) * time.Second)
+	// refresh count
```

Comment:
Can we also log large leases? I'm thinking about something like:

if l.count > largeLeaseThreshold {
klog.Infof("The object count for lease %x is large: %v", l.prevLeaseID, l.count)
}

The value from logging is that we know from our experience that the largest leases are created in tests before we start gathering metrics.

Reply:
What value of largeLeaseThreshold do you suggest?

Reply:
This should be relatively easy to change later so let's start with e.g. 5k. We will see if this makes sense.